src/test/java/com/cegeka/academy/repository/InvitationRepositoryTest.java

```diff
@@ -75,11 +76,47 @@ public void testFindByStatusWithNoResult() {
         addressRepository.saveAndFlush(address);
         Event event = TestsRepositoryUtil.createEvent("Ana are mere!", "KFC Krushers Party", true, address, user);
         eventRepository.save(event);
-        Invitation invitation1 = TestsRepositoryUtil.createInvitation("pending", "ana are mere", event, user);
+        Invitation invitation1 = TestsRepositoryUtil.createInvitation(InvitationStatus.PENDING.name(), "ana are mere", event, user);
         invitationRepository.save(invitation1);
-        Invitation invitation2 = TestsRepositoryUtil.createInvitation("pending", "maria are pere", event, user);
+        Invitation invitation2 = TestsRepositoryUtil.createInvitation(InvitationStatus.PENDING.name(), "maria are pere", event, user);
         invitationRepository.save(invitation2);
-        List<Invitation> list = invitationRepository.findByStatus("invited");
+        List<Invitation> list = invitationRepository.findByStatus(InvitationStatus.ACCEPTED.name());
         assertThat(list.size()).isEqualTo(0);
     }
+
+    @Test
+    public void testFindByUser_IdAndStatusIgnoreCase() {
```

Comment:
Imparte metoda in doua. Pare ca nu prima parte nu are legatura cu a doua.
Un test unitar ar trebui sa verifice doar un caz.